Orange/tests/test_dasktable.py

```diff
@@ -189,3 +189,8 @@ def test_str(self):
         self.assertEqual(150, len(lines))
         self.assertEqual("[[5.1, 3.5, 1.4, 0.2 | Iris-setosa],", lines[0])
         self.assertEqual(" [5.9, 3.0, 5.1, 1.8 | Iris-virginica]]", lines[-1])
+
+    def test_table_len(self):
+        iris = temp_dasktable("iris")
+        iris = iris[iris.X[:, 0] > 5.5]  # make sure length is unknown
```

Comment:
Can you also verify here that len is unknown? So instead of having a comment we should check something dask-internal.

Reply:
As a general principle it is always better to write an assertion instead of putting it in a comment.

Another option would be to make sure that len() internally computes indices.